.github/workflows/sarplus.yml

```diff
@@ -0,0 +1,168 @@
+# This workflow will run tests and do packaging for contrib/sarplus.
+#
+# Refenreces:
+#   * [GitHub Actions doc](https://docs.github.com/en/actions)
+#   * GitHub Actions workflow templates
+#       + [python package](https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml)
+#       + [python publish](https://github.com/actions/starter-workflows/blob/main/ci/python-publish.yml)
+#       + [scala](https://github.com/actions/starter-workflows/blob/main/ci/scala.yml)
+#   * [GitHub hosted runner - Ubuntu 20.04 LTS](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md)
+#   * [Azure Databirkcs runtime releases](https://docs.microsoft.com/en-us/azure/databricks/release-notes/runtime/releases)
+#   * Package publish
+#       + [Publishing package distribution releases using GitHub Actions CI/CD workflows](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/)
+#       + [pypa/gh-action-pypy-publish](https://github.com/pypa/gh-action-pypi-publish)
+
+
+name: sarplus package
+
+on:
+  push:
+    paths:
+      - contrib/sarplus/python/**
+      - contrib/sarplus/scala/**
+      - .github/workflows/sarplus.yml
+
+env:
+  PYTHON_ROOT: ${{ github.workspace }}/contrib/sarplus/python
+  SCALA_ROOT: ${{ github.workspace }}/contrib/sarplus/scala
+  SARPLUS_VERSION: 0.5.0
```

Comment:
This 0.5.0 exists in multiple locations (also in setup.py). Could you have it in a single place so that we avoid mistakes related to the version number?

Reply:
The version is moved to the file VERSION, so that Python and Scala packages share the same version.